foundation-extensions/adobe-experience-platform-assurance/README.md

```diff
@@ -404,6 +405,10 @@ class CoreStartCompletionCallback : Java.Lang.Object, IAdobeCallback
 
 The `startSession` API needs to be called to begin a Project Griffon session. When called, SDK displays a PIN authentication overlay to begin a session.
 
+{% hint style="info" %}
+Android does not require this API to be called. The AEP Assurance Android extension registers the app lifecycle handlers which automatically pick up any deep links and use them to start the session.
+{% endhint %}
+
 {% hint style="info" %}
 You may call this API when the app launches with a url \(see code snippet below for sample usage\)
 {% endhint %}
```

Comment:
Can we merge these into one single hint? e.g
{% hint style="info" %}
The AEP Assurance Android extension does not require this API to be called since it registers the app lifecycle handlers which automatically pick up any deep links and use them to start the session.
For the other platforms, you may call this API when the app launches with a url (see code snippet below for sample usage).
{% endhint %}